__tests__/articles.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api/articles/:article_id", () => {
+  it("will return 200 if it is a valid id", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  it("will return with an article object with the correct properties.", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toBeInstanceOf(Object);
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
```

Comment:
Remember that SQL assigns an id number to each article
you can also test for the types of each property -
expect(article).toHaveProperty("article_id", expect.any(Number))